src/main/java/Duke.java

```diff
@@ -1,10 +1,104 @@
+import java.util.Objects;
+import java.util.Scanner;
 public class Duke {
+    private static final String CHATBOT_NAME = "Horizon";
```

Comment:
Perhaps you can rename the file name from Duke.java to Horizon.java to make it more clear to someone veiwing your files